collectors/charts.d.plugin/libreswan/metadata.yaml

```diff
@@ -1,54 +1,118 @@
+# yamllint disable rule:line-length
+---
 meta:
   plugin_name: charts.d.plugin
   module_name: libreswan
   monitored_instance:
     name: Libreswan
-    link: ''
+    link: "https://libreswan.org/"
     categories:
-    - data-collection.vpns
-    icon_filename: 'libreswan.png'
+      - data-collection.vpns
+    icon_filename: "libreswan.png"
   related_resources:
     integrations:
       list: []
   info_provided_to_referring_integrations:
-    description: ''
-  keywords: []
+    description: ""
+  keywords:
+    - vpn
+    - libreswan
+    - network
+    - ipsec
   most_popular: false
 overview:
   data_collection:
-    metrics_description: 'Monitor Libreswan performance for optimal IPsec VPN operations. Improve your VPN operations with Netdata''s real-time metrics and built-in alerts.'
-    method_description: ''
+    metrics_description: "Monitor Libreswan performance for optimal IPsec VPN operations. Improve your VPN operations with Netdata''s real-time metrics and built-in alerts."
+    method_description: "The collector uses the `ipsec` command to collect the information it needs."
   supported_platforms:
     include: []
     exclude: []
   multi_instance: true
```

Comment:
again, here I am not sure this is multi_instance

Reply:
Yes, I think it is this one also. It will try to detect multiple tunnels if they exist, and will have charts for each one. I haven't used it, but it's what I gather from it's readme and code.

Reply:
multi-instance means multiple data collection jobs

Reply:
yeah, that's the thing, I think these just run a command in the local host and that's it